packages/es-components/src/components/containers/tabPanels/Tab.js

```diff
@@ -79,16 +79,16 @@ function Tab({
             {`${simpleName} ${announcerText}`}
           </AriaAnnouncer>
         )}
-      {name}
+      {header}
     </TabButton>
   );
 }
 
 Tab.propTypes = {
   /**
-   * The name of the tab, and the displayed value
+   * The header of the tab, and the displayed value
    */
-  name: PropTypes.node.isRequired,
+  header: PropTypes.node.isRequired,
```

Comment:
Per @jrios's [previous comment](https://github.com/WTW-IM/es-components/pull/237#issuecomment-442602849)

> Yes, this definitely is a breaking change. does the prop name change need to happen?
> 
> I would like us to try and not add any breaking changes unless absolutely necessary. If they are necessary, let's see about putting them into `next`.

Why do we need to rename this prop? From what I can tell we should be able to maintain the rest of these changes without renaming a prop and triggering a breaking change.

Reply:
This is a complete rework of how the tab panel works. Not just a prop changed. I completely overhauled how state is managed with the component